pladdy/cabby
# Migration

#### What's new
- added migrate command to cli
  - note: should probably just use a tool like https://sqitch.org/ 
  - did it this way to think through migrations
- removed prefixes from table names

#### How to test
`make test`

To verify on a VM:
```sh
make vagrant
vagrant provision --provision-with build-cabby
vagrant provision --provision-with install-cabby
curl -sk -basic -u test@cabby.com:test-password -H 'Accept: application/vnd.oasis.taxii+json' 'https://localhost:1234/taxii/' | jq .
vagrant destroy -f
```
